### PR TITLE
Document how to enable Confidential computing on CRN

### PR DESCRIPTION
**Pull request by olethanh**

### CODE REVIEW

**Review by nesitor — APPROVED** [review]

File: docs/nodes/compute/advanced/enable-confidential.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+The supported processors include the [9004 Series Processors and 8004 Series Processors](https://www.amd.com/en/products/processors/server/epyc/4th-generation-9004-and-8004-series.html#tabs-4380fde236-item-2130f0d757-tab).
+
+Note that the [4004 Series Processors do not provide SEV](https://www.amd.com/en/products/processors/server/epyc/infinity-guard.html) and are therefore not supported.
+
+> ℹ️ The 4th Generation requirement stems from security vulnerabilities discovered in SEV on Zen3 and earlier architectures.
+
+## Additional Software Requirements
+In addition to the standard software requirements, the following must be configured:
+
+* **BIOS Configuration**: SEV support must be [enabled in the BIOS](https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/tuning-guides/58207-using-sev-with-amd-epyc-processors.pdf) (refer to Section 2.1 of the document). (see Section 2.1).
```
Comment (hoh) [inline docs/nodes/compute/advanced/enable-confidential.md:17]: Markdown layout issue with MkDocs: 
image

File: docs/nodes/compute/advanced/enable-confidential.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* **Kernel and Platform Support**: The operating system kernel must support SEV. For example, Ubuntu 24.04 includes this support by default.
+* **sevctl**: The [sevctl](https://github.com/virtee/sevctl) tool must be installed. This utility is included in the aleph-vm Debian package and is installed at `/opt/sevctl`.
+* **QEMU**: QEMU must be installed on the system. `apt install cloud-image-utils qemu-utils qemu-system-x86`
+
+To verify that your system supports AMD SEV, run the following command:  `sevctl ok`
```
Comment (hoh) [inline docs/nodes/compute/advanced/enable-confidential.md:22]:
```suggestion
To verify that your system supports AMD SEV, run the following command:  `/opt/sevctl ok`
```

**Review by hoh — CHANGES_REQUESTED** [review]
Thanks !

Spotted two small issues, see my comments.

File: docs/nodes/compute/advanced/enable-confidential.md (hunk 2 above)
Comment (nesitor) [reply docs/nodes/compute/advanced/enable-confidential.md:22]: If the command is well installed, will not be needed to run from `/opt` folder

File: docs/nodes/compute/advanced/enable-confidential.md (hunk 1 above)
Comment (olethanh) [reply docs/nodes/compute/advanced/enable-confidential.md:17]: Fixed

File: docs/nodes/compute/advanced/enable-confidential.md (hunk 2 above)
Comment (olethanh) [reply docs/nodes/compute/advanced/enable-confidential.md:22]: as andres said

**Review by hoh — APPROVED** [review]

**Pull request merged**